Formula/corectl.rb

```diff
@@ -20,19 +20,22 @@ class Corectl < Formula
   def install
     ENV["GOPATH"] = buildpath
 
-    mkdir_p buildpath/"src/github.com/TheNewNormal/"
-    ln_s buildpath, buildpath/"src/github.com/TheNewNormal/#{name}"
+    path = buildpath/"src/github.com/TheNewNormal/#{name}"
+    path.install Dir["*"]
+
     Language::Go.stage_deps resources, buildpath/"src"
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
-    system "make", "corectl", *args
-    system "make", "documentation/man"
+    cd path do
+      system "make", "corectl", *args
+      system "make", "documentation/man"
```

Comment:
fwiw, here locally with ... `brew install  Formula/corectl.rb --no-clean -v -s ` i get ...
```
=> make corectl VERSION=0.5.5
fatal: Not a git repository (or any of the parent directories): .git
godep go build -o corectl -gcflags "" -ldflags " -w -s -X main.Version=0.5.5 -X main.BuildDate=2016-05-12T01:25:34WEST"
godep: WARNING: Go version (go1.6) & $GO15VENDOREXPERIMENT= wants to enable the vendor experiment, but disabling because a Godep workspace (Godeps/_workspace) exists
==> make documentation/man
fatal: Not a git repository (or any of the parent directories): .git
./corectl utils mkMan
==> Cleaning
==> Finishing up
ln -s ../Cellar/corectl/0.5.5/bin/corectl corectl
ln -s ../Cellar/corectl/0.5.5/share/cloud-init cloud-init
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl.1 corectl.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_kill.1 corectl_kill.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_load.1 corectl_load.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_ls.1 corectl_ls.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_ps.1 corectl_ps.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_pull.1 corectl_pull.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_put.1 corectl_put.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_query.1 corectl_query.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_rm.1 corectl_rm.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_run.1 corectl_run.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_ssh.1 corectl_ssh.1
ln -s ../../../Cellar/corectl/0.5.5/share/man/man1/corectl_version.1 corectl_version.1
ln -s ../Cellar/corectl/0.5.5/share/profiles profiles
==> Summary
🍺  /usr/local/Cellar/corectl/0.5.5: 17 files, 11.4M, built in 16 seconds
```
it doesn't (or shouldn't) be a missing build dep and the blob gets built (should be all that 's needed). i'm on OS X 10.11.5 Beta (15F31a) if that matters.

Anyway, going to sleep a bit. Later! (and thanks again! )